src/components/Pagination/pagination-utils.ts

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+export function getLinkDataCy(index: number, totalPages: number): string {
+  switch (index) {
+    case 0:
+      return 'prevLink';
+    case totalPages + 1:
+      return 'nextLink';
+    default:
+      return 'pageLink';
+  }
```

Comment:
The `getLinkDataCy` function could be simplified. Instead of using a switch statement, consider using a mapping object or a simple if-else construct if the logic is not too complex. This can make the code more readable and maintainable.